src/loaders/compute_tools/tool_common.py

```diff
@@ -231,11 +250,12 @@ def parallel_single_execution(self, tool_callable: Callable[[str, Path, Path, bo
                   each individual genome directory. Parser program will parse the result file in each individual genome
                   directory.
 
-        tool_callable - the callable for the tool that takes four arguments:
-            * The data ID
+        tool_callable - the callable for the tool that takes 4 arguments:
+            * The tool identifier
```

Comment:
This is always confusing to me - does this mean like `gtdb_tk` or the ID used to identify a genome to the tool? I know it's the latter but someone new to the code would be confused I think

Reply:
I will use genome_id instead of tool identifier. It's less confusing to me as well. 

Also we kinda use data ID and kbase ID interchangeably. Any preference here?

Reply:
For data ID, let's stick with data ID for now as I suspect we'll want to change the `kbase_id` in the final output to something other than the UPA at some point and it'd be good to have different names.

Reply:
I'm not really a fan of `genome_id` either because it doesn't really describe the purpose of the ID and why there's a data_id and a genome_id that both refer to the same thing? 

Maybe `tool_safe_data_id`? That's a bit long but more understandable

Reply:
:+1: